docs/source/_static/data_formatting_examples/format_iss_breast_data.py

```diff
@@ -59,18 +58,36 @@ def round_dict(self):
         return round_dict
 
     def get_tile(self, fov: int, r: int, ch: int, z: int) -> FetchedTile:
-        filename = 'slideA_{}_{}_{}.TIF'.format(str(fov + 1),
-                                                self.round_dict[r],
-                                                self.ch_dict[ch]
-                                                )
+
+        # get filepath
+        fov_ = str(fov + 1)
+        round_ = self.round_dict[r]
+        ch_ = self.ch_dict[ch]
+        filename = f"slideA_{fov_}_{round_}_{ch_}.TIF"
         file_path = os.path.join(self.input_dir, filename)
-        return IssCroppedBreastTile(file_path)
+
+        # get coordinates
+        coordinates = {
+            Coordinates.X: (
+                self.coordinates_df.loc[fov, "x_min"],
+                self.coordinates_df.loc[fov, "x_max"]
+            ),
+            Coordinates.Y: (
+                self.coordinates_df.loc[fov, "y_min"],
+                self.coordinates_df.loc[fov, "y_max"]
+            ),
+            Coordinates.Z: 0
```

Comment:
Does this experiment have multiple Z layers?  If not, I think you can just omit this.